src/Couchbase.Lite.TestCoverage/Couchbase.Lite.TestCoverage.sln

```diff
@@ -15,9 +15,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Couchbase.Lite", "..\Couchb
 EndProject
 Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "LiteCore.Shared", "..\LiteCore\src\LiteCore.Shared\LiteCore.Shared.shproj", "{C37DC024-8762-4BE4-9034-B2B5970C652C}"
 EndProject
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "Couchbase.Lite.Shared", "..\Couchbase.Lite.Shared\Couchbase.Lite.Shared.shproj", "{9B92B9D6-5DD3-477A-883B-1B40D5DEEF3B}"
+EndProject
 Global
```

Comment:
Unrelated to this PR

Reply:
Still unrelated to this PR

Reply:
somehow, it's just updating now..

Reply:
but I thought you want me to remove it because it's unrelated to the feature?